package.json

```diff
@@ -54,6 +53,7 @@
     "last 3 edge versions"
   ],
   "devDependencies": {
+    "@rollup/plugin-node-resolve": "^15.1.0",
```

Comment:
These 3 dev dependencies are used to get the `@hooks`, `@components`, etc... mappings working. This one runs on the prod build, the eslint ones run in eslint.